### DISCUSSION TITLE
[Question] How to sign Doge Coin transaction?

### DISCUSSION THREAD

**Discussion by Raden-Hor** [trustwallet/wallet-core#1976] [Q&A]
Does anyone know how to sign DOGE Coin transaction in wallet core android?

**Comment by optout21** [comment]
DOGE is similar to Bitcoin, I recommend:
https://github.com/trustwallet/wallet-core/tree/master/samples/android

**Reply by siddikbitxpe** [reply]
val utxoTxId = "050d00e2e18ef13969606f1ceee290d3f49bd940684ce39898159352952b8ce2".hexStringToByteArray();

how can I 050d00e2e18ef13969606f1ceee290d3f49bd940684ce39898159352952b8ce2 this value

**Comment by siddikbitxpe** [comment]
val utxoTxId = "050d00e2e18ef13969606f1ceee290d3f49bd940684ce39898159352952b8ce2".hexStringToByteArray();

how we can get 050d00e2e18ef13969606f1ceee290d3f49bd940684ce39898159352952b8ce2 this value

**Reply by Raden-Hor** [reply]
You can use https://www.npmjs.com/package/blockbook-client with node rpc doge1.trezor.io

**Reply by gabriellahayden** [reply]
Are you all still having issues signing doge coin transaction?

**Reply by siddikbitxpe** [reply]
I want to use for android  and please check 
https://github.com/trustwallet/wallet-core/blob/dad35d9a32c6935ef5853c7fddb5f00485c2438f/samples/android/app/src/main/java/com/trust/walletcore/example/MainActivity.kt#L68

**Reply by radenhor18** [reply]
@siddikbitxpe u can use this rest api to get utxoTxId https://doge1.trezor.io/api/v1/utxo/<dogecoin_address>
ref: https://github.com/trezor/blockbook/blob/master/docs/api.md#rest-api

**Comment by gabriellahayden** [comment]
Are you all still having issues signing doge coin transaction?
